nemo/collections/common/data/lhotse/nemo_adapters.py

```diff
@@ -172,30 +182,30 @@ def __init__(
         text_field: str = "text",
         lang_field: str = "lang",
     ) -> None:
-        def strip_pipe(p):
-            if isinstance(p, str):
-                if p.startswith("pipe:"):
-                    p = p[5:]
-                return Path(p)
-            return p
-
         self.shard_id_to_manifest: dict[int, Iterable[dict]]
         self.paths = expand_sharded_filepaths(manifest_path)
         if len(self.paths) == 1:
             self.source = LazyJsonlIterator(self.paths[0])
             self.shard_id_to_manifest = groupby("shard_id", self.source)
         else:
-            pattern = re.compile(r".+_(\d+)\.jsonl?(?:.gz)?")
+            json_pattern = re.compile(r"manifest[^/]*_(\d+)[^/]*\.json")
```

Comment:
nit: may be add `jsonl`

Reply:
it's used in `re.search` so it will match as if it's `...json.*`